src/analyses/basePriv.mli

```diff
@@ -1,5 +1,6 @@
 open Cil
 (* Cannot use local module substitutions because ppx_import is still stuck at 4.07 AST: https://github.com/ocaml-ppx/ppx_import/issues/50#issuecomment-775817579. *)
+(* TODO: try again, because ppx_import is now removed *)
```

Comment:
Just a comment to make this TODO visible in the PR Conversation...

Reply:
I might try this again after #280 since it'll upgrade a bunch of other ppx dependencies too.